consensus/ConsensusFileGenerator.py

```diff
@@ -21,9 +21,14 @@ def __init__(self, data, tables):
         comments_table = tables['comments_table']
         self.consensus_data = data['consensus_data']
         self.lab_classifications = data['lab_classifications']
-        self.history = data['history']
+        self.history = data['history']['history']
+        self.alternative_history = data['history']['alternative']
         self.consensus_table_file_name = consensus_table
         self.comments_table_file_name = comments_table
+        self.incorrect_variant_history_file_name = incorrect_variant_history_file
+        if self.incorrect_variant_history_file_name:
+            incorrect_history_file = open(self.incorrect_variant_history_file_name, 'w')
```

Comment:
what is the purpose of opening and immediately closing a file?

Reply:
By opening a file in the "w" mode, the file will be emptied, this way I make sure the file I append to later, will be empty when I do so.